runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -91,11 +101,16 @@ ClientSessionData::updateTimeOfLastAccess()
 void
 ClientSessionData::processUnloadedClasses(JITServer::ServerStream *stream, const std::vector<TR_OpaqueClassBlock*> &classes)
    {
+   const uint32_t numOfUnloadedClasses = (uint32_t)classes.size();
```

Comment:
Any particular reason you needed a cast here?

Reply:
No particular reason. Will remove the cast.